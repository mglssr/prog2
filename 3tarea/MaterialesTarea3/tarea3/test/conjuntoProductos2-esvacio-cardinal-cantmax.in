crearConjuntoProductos 2
esVacioConjuntoProductos
cardinalConjuntoProductos
cantMaxConjuntoProductos
insertarConjuntoProductos 2
cardinalConjuntoProductos
cardinalConjuntoProductos
esVacioConjuntoProductos
cardinalConjuntoProductos
cantMaxConjuntoProductos
insertarConjuntoProductos 1
cardinalConjuntoProductos
cantMaxConjuntoProductos
liberarConjuntoProductos
Fin
