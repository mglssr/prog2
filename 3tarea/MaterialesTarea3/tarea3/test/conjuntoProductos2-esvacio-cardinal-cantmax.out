Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> El conjunto productos fue creado en forma correcta.
2> El conjunto de productos es vacio.
3> El cardinal del conjunto es: 0.
4> La cantidad maxima del conjunto es: 2.
5> Se ejecuto 'insertarTConjuntoProductos' con producto de id 2 exitosamente.
6> El cardinal del conjunto es: 0.
7> El cardinal del conjunto es: 0.
8> El conjunto de productos es vacio.
9> El cardinal del conjunto es: 0.
10> La cantidad maxima del conjunto es: 2.
11> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
12> El cardinal del conjunto es: 1.
13> La cantidad maxima del conjunto es: 2.
14> Conjunto de productos liberado exitosamente.
15> Fin.
